conf/messages.en

```diff
@@ -1555,4 +1555,51 @@ unallocatedEmployerCYA.section.hide = amount of unallocated employer contributio
 
 removeUnallocatedAmount.title = Are you sure you want to remove the unallocated employer contributions?
 removeUnallocatedAmount.heading = Are you sure you want to remove the unallocated employer contributions of {0}?
-removeUnallocatedAmount.error.required = Select yes if you are sure you want to remove the unallocated employer contributions
\ No newline at end of file
+removeUnallocatedAmount.error.required = Select yes if you are sure you want to remove the unallocated employer contributions
+
+landOrPropertyPostcodeLookup.title = What is the postcode of the land or property?
+landOrPropertyPostcodeLookup.heading = What is the postcode of the land or property?
+landOrPropertyPostcodeLookup.lookup.label = Postcode
+landOrPropertyPostcodeLookup.filter.label = Property name or number (optional)
+landOrPropertyPostcodeLookup.paragraph = You can
+landOrPropertyPostcodeLookup.paragraph.link = enter the address manually.
+landOrPropertyPostcodeLookup.postcode.error.required = Enter the postcode of the land or property
+landOrPropertyPostcodeLookup.postcode.error.notFound = Postcode not found. Try another postcode or enter the address manually
+landOrPropertyPostcodeLookup.postcode.error.invalid.characters = Land or property postcode must only include letters a to z and numbers 0 to 9
+landOrPropertyPostcodeLookup.postcode.error.invalid.format = Enter the postcode of the land or property in the right format, like AB1 1AB
+landOrPropertyPostcodeLookup.filter.error.required = Enter the postcode of the land or property
+landOrPropertyPostcodeLookup.filter.error.invalid = Property name or number must only include letters a to z, numbers 0 to 9, apostrophes, ‘atʼ signs (@), forward slashes, full stops, hyphens, speech marks and spaces
+landOrPropertyPostcodeLookup.filter.error.max = Property name or number must be 35 characters or fewer
+
+landPropertyAddressResults.title = What is the address of the land or property?
+landPropertyAddressResults.heading = What is the address of the land or property?
+landPropertyAddressResults.description = Select the address from the list, or
+landPropertyAddressResults.description.link = enter the address manually.
+landPropertyAddressResults.error.required = Select the address of the land or property
+
+landPropertyAddressManual.title = What is the address of the land or property?
+landPropertyAddressManual.heading = What is the address of the land or property?
+landPropertyAddressManual.field1.label = Address line 1
+landPropertyAddressManual.field1.error.required = Enter the first line of the address
+landPropertyAddressManual.field1.error.invalid = First line of address must only include letters a to z, numbers 0 to 9, apostrophes, ‘atʼ signs (@), commas, forward slashes, full stops, hyphens, speech marks and spaces
+landPropertyAddressManual.field1.error.max = First line of the address must be 35 characters or fewer
+landPropertyAddressManual.field2.label = Address line 2 (optional)
```

Comment:
Mandatory in ETMP

Reply:
Will change 👍

Reply:
leaving it as is as per the slack discussion https://hmrcdigital.slack.com/archives/C042149LQFN/p1699362656089599